tests/topotests/ospf_multi_vrf_bgp_route_leak/r2/zebra-vrf-ray.txt

```diff
@@ -1,9 +1,9 @@
 VRF ray:
 B   10.0.1.0/24 [20/20] via 10.0.20.1, r2-eth1 (vrf default) inactive, weight 1, XX:XX:XX
-B   10.0.2.0/24 [20/0] is directly connected, r2-eth0 (vrf default) inactive, weight 1, XX:XX:XX
+B   10.0.2.0/24 [20/0] is directly connected, lo (vrf default) inactive, weight 1, XX:XX:XX
 B>* 10.0.3.0/24 [20/20] via 10.0.20.1, r2-eth1 (vrf default), weight 1, XX:XX:XX
 O>* 10.0.4.0/24 [110/20] via 10.0.40.4, r2-eth2, weight 1, XX:XX:XX
-B   10.0.20.0/24 [20/0] is directly connected, r2-eth1 (vrf default) inactive, weight 1, XX:XX:XX
+B   10.0.20.0/24 [20/0] is directly connected, lo (vrf default) inactive, weight 1, XX:XX:XX
```

Comment:
Why is this loopback? Is this sort of VRF's loopback that will be resolved to r2-eth1 or how it's supposed to work here?

Reply:
I have added a commit "topotests: test leak from default vrf"

VRF interfaces are the equivalent of the lo interface for the VRFs.

In this example, if the route to 10.0.20.0/24 were active, it would not be operational with r2-eth1 interface nexthop because r2-eth1 is unknown from vrf ray. It would be operational with lo instead. lo would reroute to r2-eth1 using the default VRF routing table